src/components/Disputes/DisputeCurrentRuling.js

```diff
@@ -116,14 +117,15 @@ function DisputeCurrentRuling({ dispute }) {
                       margin-right: ${0.5 * GU}px;
                     `}
                   >
-                    Your voting weight
+                    {showMyWeight ? 'Your v' : 'V'}oting weight
```

Comment:
Haha what do you think about just having two different strings?

Better to not be too clever, and it's also generally easier for collecting strings for internationalization :)

Reply:
Haha yes sure!